src/test/main.test.js

```diff
@@ -33,7 +33,7 @@ assert.rejects = assert.rejects || (async (p, e) => {
 
 
 const ethereum_rpc = 'http://127.0.0.1:8545';
-const contract_address = '0x2Bc0589Fc40F21254407966B7f39654a05a7CD34';
+const contract_address = '0x4A63cbeD58d96397631A6Bede32beBae2030c152';
```

Comment:
What is the reason for this change?

Reply:
I have to refresh the contract address every time I change my local testing setup. I've been meaning to change it for a while. I've added a commit that creates a connection_config.json file and .gitignores it so that these changes are not version-controlled in the future.

Reply:
Adding it along with a .gitignore entry results in it still showing as a diff, since the file still exists in git and is tracked so this only moves the issue.

Reply:
You're correct. I added another commit to remove the .gitignore. Now connection_config.json can be manually omitted from the staging area or you can use a command like `git update-index --assume-unchanged connection_config.json`.